src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerCorporateValidatorTest.java

```diff
@@ -72,12 +72,12 @@ void testErrorReportedWhenNameFieldIsNull() {
 
     @Test
     void testErrorReportedWhenNameFieldExceedsMaxLength() {
-        beneficialOwnerCorporateDtoList.get(0).setName(StringUtils.repeat("A", 51));
+        beneficialOwnerCorporateDtoList.get(0).setName(StringUtils.repeat("A", 161));
```

Comment:
for the tests, could you add a test to check that 160 is accepted (no errors). Someone could be able to change the max length value in the validator to eg. 20, and tests would probably still pass with just the 161 check.